src/scss/_main.scss

```
@@ -1011,6 +1017,18 @@ nav.bd-links {
             font-size: 0.9em;
         }
         font-size: 1em;
+        position: relative;
+    }
+
+    .fa-chevron-up, .fa-chevron-down {
+        position: absolute;
+        font-size: 0.75em;
+        right: 1rem; // gives the same padding as 1.25em
+        top: 0.5rem;
```

```suggestion
        right: 0.5em;
        top: 0;
        padding: 0.5em;
```

Let's add a bit of padding and make the potential area you need to click bigger.